components/StaLogo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components/native';
+import StaLogoUrl from '../images/RoundAvatar.png'
```

Comment:
suggest we use SVG

Reply:
They are getting a UX designer but will use these in the mean time